data/ATAC_JV_adipose/cv/equal_labels/testFolds/1/randomTrainOrderings/1/models/iregnet_log_fixed/predictions.csv

```diff
@@ -0,0 +1,131 @@
+sequenceID,V1
```

Comment:
this is a good start but please use a more informative column name instead of V1. for example typically I use pred.log.penalty or pred.log.lambda